net-ftp/lftp/lftp-4.6.5.ebuild

```diff
@@ -13,13 +13,11 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~mips ppc ppc64 ~s390 sparc x86 ~sparc-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
```

Comment:
Adding this to a stable ebuild is just not going to fly.  We can add this to 4.7.0 and/or 4.7.1 but not 4.6.5